docs/mint.json

```diff
@@ -200,6 +201,22 @@
       "group": "About",
       "pages": ["pages/v2/crm-v2/about"]
     },
+    {
+      "group": "About",
+      "pages": ["pages/chat/about"]
+    },
+    {
+      "group": "About",
+      "pages": ["pages/dialer/about"]
+    },
+    {
```

Comment:
What's the difference between having multiple objects in the array that are of the same group vs bunching all the urls into the `pages` array?

Reply:
It changes the layout, so if you nest it, it will appear nested in the UI